charmhelpers/core/sysctl.py

```diff
@@ -62,4 +64,11 @@ def create(sysctl_dict, sysctl_file, ignore=False):
     if ignore:
         call.append("-e")
 
-    check_call(call)
+    try:
+        check_call(call)
+    except CalledProcessError as e:
+        if is_container():
+            log("Error setting some systcl keys in this container: {}".format(e.output),
```

Comment:
typo - sysctl (not systcl) and please can we log this as a warning - its not fatal so it not an error.